ChangeLog

```diff
@@ -1,3 +1,270 @@
+	- ... and many documentation improvements
+	- ... and many portability and compatibility fixes
+	- ... and many code cleanups
+	- ... and improvements to unit tests
+	- ... and some error page translation improvements
+
+Changes in squid-6.13 (31 Jan 2025):
+
+	- Bug 5352: Do not get stuck when RESPMOD is slower than read(2)
+	- Bug 5405: Large uploads fill request buffer and die
+	- Bug 5093: List http_port params that https_port/ftp_port lack
+	- Bug 5311: clarify configuration byte units
+	- Bug 5091: document that changes to workers require restart
+	- Bug 5481: Fix GCC v14 build [-Wmaybe-uninitialized]
+	- Nil request dereference in ACLExtUser and SourceDomainCheck ACLs
+	- Fix GCC v14 [-Wanalyzer-null-dereference] warnings in Kerberos
+	- Clarify --enable-ecap failure on missing shared library support
+	- Fix syntax error in configure.ac
+	- Remove GNU'ism in release notes Makefile
+	- Annotate PoolMalloc memory in valgrind builds
+	- Fix systemd startup sequence to require active Local Filesystem
+	- Display Linux variant at ./configure time
+	- Refactor peerRefreshDNS() to clarify its (void*)1 logic
+	- Portability: remove explicit check for libdl
+	- ext_time_quota_acl: remove -l option
+	- ... and some documentation updates
+	- ... and some CI updates
+
+Changes in squid-6.12 (12 Oct 2024):
+
+	- Fix validation of Digest auth header parameters
+	- Fix build with libtool v2.4 + automake v1.17
+	- Use git to extract default build-info when enabled
+	- ... and a minor source-maintenance portability fix
+
+Changes in squid-6.11 (15 Sep 2024):
+
+	- Fix Kerberos detection when cross-compiling
+	- Improve robustness of DNS code on reconfigure
+	- Prevent slow memory leak in TCP DNS queries
+	- Improve errors emitted when invalid ACLs are parsed
+	- Remove some unused code and obsolete compatiblity hacks
+
+Changes in squid-6.10 (10 Jun 2024):
+
+	- Bug 5378: type mismatch in libTrie
+	- Disable ESI in default build
+	- Fix build with clang v18 [-Wvla-cxx-extension]
+	- ... and some unit test improvements
+	- ... and some documentation improvements
+
+Changes in squid-6.9 (7 Apr 2024):
+
+	- Regression Bug 5349: basic_nis_auth build error: unterminated #ifndef
+	- Bug 5069: Keep listening after getsockname() error
+	- Bug 5360: FwdState::noteDestinationsEnd() assertion "err"
+	- Reduce stale errno usage
+	- Plug memory leak in handling cache manager requests
+	- Fix error: template-id not allowed for constructor in C++20
+	- Improve release packaging automation
+
+Changes in squid-6.8 (4 Mar 2024):
+
+	- Bug 5344: mgr:config segfaults without logformat
+	- Bug 5343: Fix build on GCC 14
+	- Fix infinite recursion when parsing malformed HTTP chunks
+	- Fix max-stale in default refresh_pattern
+	- Fix marking of problematic cached IP addresses
+	- Improved portability to MacOS
+	- ... and some documentation improvements
+
 Changes in squid-6.7 (4 Feb 2024)
 
 	- Bug 5337: workaround for crash on startup if -a option is used
```

Comment:
Please consider adding v5.10 section as well!

Reply:
There is no SQUID_5_10 tag in the repository :\

Reply:
But there is v5.10 ChangeLog section in v5 branch. That is enough as far as this PR is concerned.

Reply:
Added the ChangeLog session but we don't have a release. We need to identify the commit to hang it from

Reply:
> Added the ChangeLog session 

FYI: This PR still does not have a v5.10 section added. The last pushed commit is 9cb207b.
 

> but we don't have a release. We need to identify the commit to hang it from

I suggest commit a254dfea, but it is not my call to make. This additional v5.10 work can be done later, of course.